protocols/client/client.go

```diff
@@ -31,6 +32,7 @@ const (
 )
 
 var defaultDialTimeout = 15 * time.Second
+var defaultCloseTimeout = 5 * time.Second
```

Comment:
have you tested this change with vsocks? see https://github.com/kata-containers/agent/issues/324 and https://github.com/kata-containers/agent/commit/93fe84e041b8895e68d268246a06c5a9a1395f3c

Reply:
why you are reducing the timeout again?

Reply:
This is close timeout not dial timeout. Why would a guest take more than 5 seconds to close a connection?

And it is only used by the yamux client. So only kata-runtime and kata-proxy will use it.

Reply:
make sense, thanks